fix(windows): fix command bug in update elevated task (fix #11216) (#11217)
tauri-apps/tauri: .changes/fix-elevated-update-cmd.md

---

> "tauri-bundler": patch:fix

Reviewer: ```suggestion
"tauri-bundler": patch:bug
```

---

> Fixed Command arguments for Windows -> msi -> elevatedUpdateTask. This works with spaces in project name on windows

Reviewer: ```suggestion
On Windows, fixed command arguments for `bundle -> windows -> msi -> elevatedUpdateTask`. to work with spaces in `productName`
```
